AgileCoffeeShop/CoffeeShop.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace AgileCoffeeShop
+{
+    public class CoffeeShop
+    {
+        #region Members
+        public string Name { get; set; }
+        public Coordinate Coordinate { get; set; }
+        #endregion
+
+        public CoffeeShop ()
+        {
+            Name = string.Empty;
+            Coordinate = new Coordinate();
+            
```

Comment:
Normally there are tools which are doing your cleanup on the code, but it's kind of disruptive to make code review and see extra empty spaces. Bad habits :)) 
E.g: FxCop : https://help.sider.review/tools/csharp/fxcop

Reply:
I haven't used any tools like that, I'm a junior and I'still learning, but I agree that empty spaces can be disturbing.

Reply:
Maybe there was a misunderstanding there, sorry about that. 
I wasn't expecting you to use tools like that before, was more like for knowing about them in the future